### PR TITLE
Qwen35 397b GRPO ruleset and rcps

### PR DESCRIPTION
**Pull request by jepio**
The ruleset requires evaluation to start at a fixed step and happen every step after that until target accuracy is reached.

GBS | val start step | val start samples 
-|-|-
256 | 18 | 4608
512 | 10 | 5120
1024 | 7 | 7168

### CODE REVIEW

File: mlperf_logging/compliance_checker/training_6.1.0/closed_qwen35_397b_grpo.yaml [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+- KEY:
+    NAME:  opt_base_learning_rate
+    REQ:   EXACTLY_ONE
+    CHECK: " abs(v['value'] - 1.0e-6 * (s['global_batch_size'] / 256) ** 0.5) < 1e-9 "
+
+- KEY:
+    NAME:  opt_end_learning_rate
+    REQ:   EXACTLY_ONE
+    CHECK: " abs(v['value'] - 1.0e-6 * (s['global_batch_size'] / 256) ** 0.5) < 1e-9 "
```
Comment (ShriyaRishab) [inline mlperf_logging/compliance_checker/training_6.1.0/closed_qwen35_397b_grpo.yaml]: If base LR and end LR are the same and the LR follows constant schedule, why do we need two keys?

File: mlperf_logging/compliance_checker/training_6.1.0/closed_qwen35_397b_grpo.yaml [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+- KEY:
+    NAME:  opt_learning_rate_warmup_steps
+    REQ:   EXACTLY_ONE
+    CHECK: " v['value'] == 0 "
+
+- KEY:
+    NAME:  opt_learning_rate_decay_steps
+    REQ:   EXACTLY_ONE
+    CHECK: " v['value'] == 100000 "
```
Comment (ShriyaRishab) [inline mlperf_logging/compliance_checker/training_6.1.0/closed_qwen35_397b_grpo.yaml]: Why does decay steps exist for constant LR?

Comment (jepio) [reply mlperf_logging/compliance_checker/training_6.1.0/closed_qwen35_397b_grpo.yaml]: we don't, you're right, removed it

File: mlperf_logging/compliance_checker/training_6.1.0/closed_qwen35_397b_grpo.yaml (hunk 1 above)
Comment (jepio) [reply mlperf_logging/compliance_checker/training_6.1.0/closed_qwen35_397b_grpo.yaml]: `opt_end_learning_rate` is unnecessary if we assert that the schedule is `constant` - removed this

**Review by ShriyaRishab — APPROVED** [review]
